src/models/methods/unsubscribe.ts

```diff
@@ -0,0 +1,23 @@
+import { Currency, StreamType } from "../common";
+import { BaseRequest, BaseResponse } from "./baseMethod";
+
+interface Book {
+    taker_gets: Currency
+    taker_pays: Currency
+    both?: boolean
+}
+  
+export interface UnsubscribeRequest extends BaseRequest {
+    command: "unsubscribe"
+    streams?: StreamType[]
+    accounts?: string[]
+    accounts_proposed?: string[]
+    books?: Book[]
+    url?: string
+    url_username?: string
+    url_password?: string
+}
```

Comment:
It seems the url parameters ("url", "url_username", and "url_password") are deprecated and may be removed without notice: https://xrpl.org/unsubscribe.html#request-format